install/gcp/scripts/initialize_forseti_services.sh

```diff
@@ -85,6 +85,7 @@ User=ubuntu
 Restart=always
 RestartSec=3
 ExecStart=$FORSETI_COMMAND
+Environment="POLICY_LIBRARY_HOME=${POLICY_LIBRARY_HOME}"
```

Comment:
This seemed like the easiest way to pass in the path of the Policy Library to Forseti. We already have an env variable and did not want to add additional command line arguments.